generate_shuffle_tables.py

```
@@ -0,0 +1,198 @@
+            if code == 0:
+                total_len += 0  # 0 bytes
+            elif code == 1:
+                total_len += 1  # 1 byte
+            elif code == 2:
+                total_len += 2  # 2 bytes
+            elif code == 3:
+                total_len += 4  # 4 bytes
+        lengths.append(total_len)
+    return lengths
+
+def generate_length_table_1234():
```

Ruff (0.12.2)

Missing return type annotation for public function `generate_length_table_1234`

(ANN201)